content/400-reference/100-system-requirements.mdx

```diff
@@ -21,7 +33,48 @@ Notes:
 
 See also: [Supported database versions](database-reference/supported-databases)
 
-</TopBlock>
+### Operating systems
+
+Prisma is supported on MacOS, Windows and most Linux distributions.
+
+#### Linux runtime dependencies
+
+Prisma requires the following system libraries installed to work:
+
+- OpenSSL 1.0.x (except Alpine Linux), 1.1.x or 3.x
```

Comment:
Why is the `(except Alpine Linux)` in the middle of the versions here? Does that only apply to `1.0.x` somehow?

Reply:
No build for Alpine Linux relies on `openssl-1.0.x`

Reply:
In other words, we only support `1.1.x` and `3.0.x` in Alpine, regardless of the architecture